ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -33,10 +33,15 @@ import useAlerts from '../../../../../hooks/useAlerts';
 import { Alert } from '../../../../../ducks/confirm-alerts/confirm-alerts';
 
 export type AlertModalProps = {
-  /** The owner ID of the relevant alert from the `confirmAlerts` reducer. */
-  ownerId: string;
   /** The unique key representing the specific alert field */
   alertKey: string;
+  /**
+   * The navigation component passed when more exists more than one alert.
```

Comment:
We're still coupling to the navigation in the component but conceptually we don't need to since that's a decision made by the `MultipleAlertModal`.